control-plane/api-gateway/controllers/gateway_controller.go

```diff
@@ -782,6 +795,52 @@ func (c *GatewayController) getRelatedHTTPRoutes(ctx context.Context, gateway ty
 	return list.Items, nil
 }
 
+func (c *GatewayController) getExternalFiltersForHTTPRoute(ctx context.Context, route gwv1beta1.HTTPRoute, resources *common.ResourceMap) ([]interface{}, error) {
+	var externalFilters []interface{}
+	for _, rule := range route.Spec.Rules {
+		for _, filter := range rule.Filters {
+			c.Log.Info("\n\n filter iterating")
```

Comment:
Is the extra logging in this function intended to stick around?

Reply:
Nope, cleaned it up